code/datums/elements/food/processable.dm

```diff
@@ -32,7 +32,6 @@
 	SIGNAL_HANDLER
 
 	mutable_recipes += list(list(TOOL_PROCESSING_RESULT = result_atom_type, TOOL_PROCESSING_AMOUNT = amount_created, TOOL_PROCESSING_TIME = time_to_process))
-	return COMPONENT_NO_AFTERATTACK
```

Comment:
funny note: this should actually be COMPONENT_BLOCK_TOOL_ATTACK

Reply:
Then add it

Reply:
see comment

Reply:
what comment